ui/src/router/activity_log/events_list/EventsList.js

```diff
@@ -19,33 +19,36 @@ export const EventsList = ({ events, status }) => {
       case "error":
         return "alert";
       default:
-        return undefined;
+        return "dot";
     }
   };
 
   return (
     <Fragment>
-      {events.map((event, idx) => (
-        <EuiComment
-          key={`event-${idx}`}
-          username={badge(event)}
-          timelineIcon={timelineIcon(event)}
-          type="update"
-          timestamp={<DateFromNow date={event.created_at} size={"s"} />}>
-          <ExpandableContainer
-            maxCollapsedHeight={43}
-            toggleKind="text"
-            isScrollable={false}>
-            <span className="activityLogEvent">{event.message}</span>
-          </ExpandableContainer>
-        </EuiComment>
-      ))}
-      {status === Status.PENDING && (
-        <EuiComment
-          username={<PulseLoader size={6} color={infoColor} />}
-          type="update"
-        />
-      )}
+      <EuiCommentList>
+        {events.map((event, idx) => (
+          <EuiComment
+            key={`event-${idx}`}
+            username={badge(event)}
+            timelineAvatarAriaLabel="System"
```

Comment:
I don't think we need an `aria-label` here since its not an interactive icon anyway.

Reply:
Just removed that :)